src/mrpro/algorithms/csm/iterative_walsh.py

```diff
@@ -45,6 +40,14 @@ def iterative_walsh(
         width of the smoothing filter
     power_iterations
         number of iterations used to determine dominant eigenvector
+
+    References
+    ----------
+    .. [1] https://github.com/ismrmrd/ismrmrd-python-tools
+    .. [2] Daval-Frerot G, Ciuciu P (2022) Iterative static field map estimation for off-resonance correction in
```

Comment:
would be good to use the same style as in Rotation.py

Reply:
There are actually 2 different styles within Rotation.py 
One in lines 367-369, the other in lines 640-643

I stuck to the latter throughout my changes